test/templates/unit_test.tpl

```diff
@@ -92,7 +92,7 @@ jobs:
         sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 467B942D3A79BD29
 
         # mysql80
-        wget -c https://dev.mysql.com/get/mysql-apt-config_0.8.14-1_all.deb
+        wget -c https://dev.mysql.com/get/mysql-apt-config_0.8.3-1_all.deb
```

Comment:
Correct me if i am wrong, but shouldn't this be `mysql-apt-config_0.8.30-1_all.deb` or `mysql-apt-config_0.8.31-1_all.deb` ?

Reply:
This is the apt-config package, not the actual package. The latest version available for it is `0.8.20-1`, and that is what we should use

Reply:
Okay, makes sense!